lib/ansible/plugins/callback/cloudwatch_logs.py

```diff
@@ -0,0 +1,351 @@
+from __future__ import (absolute_import, division, print_function)
+
+import time
+import uuid
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: cloudwatch_logs
+    callback_type: notification
+    requirements:
+      - whitelist in configuration
+      - boto3 (python library)
+    short_description: Sends play events AWS CloudWatch Logs
+    version_added: "2.7"
+    description:
+        - This is an ansible callback plugin that sends status updates to CloudWatch Logs during playbook execution.
+    options:
+      log_group_name:
+        required: True
+        description: group name the logs are written against
+        env:
+          - name: CLOUDWATCH_LOGS_LOG_GROUP_NAME
+        ini:
+          - section: callback_cloudwatch_logs
+            key: log_group_name
+
+      log_stream_name:
+        required: True
+        description: stream name the logs are written against
+        env:
+          - name: CLOUDWATCH_LOGS_LOG_STREAM_NAME
+        ini:
+          - section: callback_cloudwatch_logs
+            key: log_stream_name
+
+      aws_region_name:
+        required: False
+        description: aws region to use
+        env:
+          - name: CLOUDWATCH_LOGS_AWS_REGION_NAME
+        ini:
+          - section: callback_cloudwatch_logs
+            key: aws_region_name
+
+      aws_access_key_id:
+        required: False
+        description: aws credentials
+        env:
+          - name: CLOUDWATCH_LOGS_AWS_ACCESS_KEY_ID
+        ini:
+          - section: callback_cloudwatch_logs
+            key: aws_access_key_id
+
+      aws_secret_access_key:
+        required: False
+        description: aws credentials
+        env:
+          - name: CLOUDWATCH_LOGS_AWS_SECRET_ACCESS_KEY
+        ini:
+          - section: callback_cloudwatch_logs
+            key: aws_secret_access_key
+
+      aws_kms_key_id:
+        required: False
+        description: id of the kms key to encrypt the logs with
+        env:
+          - name: CLOUDWATCH_LOGS_AWS_KMS_KEY_ID
+        ini:
+          - section: callback_cloudwatch_logs
+            key: aws_kms_key_id
+'''
+
+import getpass
+import os
+import boto3
```

Comment:
A try/except ImportError block is needed around all imports that are not from the Python standard library.

Take a look at other callback plugins to see how that is handled.